parsers/src/test/resources/xcsp/instances.csv

```diff
@@ -3,14 +3,14 @@ basics;Allergy.xml.lzma;1;_;1;0
 basics;AllInterval-005.xml.lzma;1;_;13;9
 basics;Auction-cnt-example_c18.xml.lzma;2;54;7;4
 basics;Auction-sum-example_c18.xml.lzma;2;54;6;3
-basics;Bacp-m1-06_c18.xml.lzma;3;10;38111;37700
```

Comment:
shouldn't we add the time ? 
It should be compared with care but still it would be nice to have an idea if it is faster or slower

Reply:
Honestly, I won't add this for the reason described here (https://buildjet.com/for-github-actions/blog/why-github-actions-is-so-slow#what-hardware-is-actually-github-actions-using).
In two words,  you cannot select the hardware the VM is running on.
This explains why Minizinc tests often fail on GH even though they work on my computer.

It's something I've been wanting to check for a long time now, but I can't find the right way to do it.

Reply:
I agree that GitHub test runtime is hard to trust, but this file is updated locally (i.e. on your computer), no ?
Even if it is not perfect (various contributors may have several machines), it may be better than nothing...

Reply:
Such artifact already exists. For instance, https://github.com/chocoteam/choco-solver/actions/runs/5766875782 you can get the artifact which contains 2 files, one for xcsp and the other for mzn

Reply:
Maybe the question should be: how to test performance regression?